pkgs/by-name/re/rerun/package.nix

```diff
@@ -99,6 +99,24 @@ rustPlatform.buildRustPackage (finalAttrs: {
     nasm
   ];
 
+  # NOTE: Without setting these environment variables the web-viewer
+  # preBuild step uses the nix wrapped CC which doesn't support
+  # multiple targets including wasm32-unknown-unknown. These are taken
+  # from the following issue discussion in the rust ring crate:
+  # https://github.com/briansmith/ring/discussions/2581#discussioncomment-14096969
+  env =
+    let
+      inherit (llvmPackages) clang-unwrapped;
+      major-version = builtins.head (builtins.splitVersion clang-unwrapped.version);
+      # resource dir + builtins from the unwrapped clang
+      resourceDir = "${lib.getLib clang-unwrapped}/lib/clang/${major-version}";
```

Comment:
`pkgs/development/compilers/llvm/common/default.nix` seems to feature the same logic, but I'm not immediately sure how to correctly consume it; probably there's just another wrapper we could refer to

Either way, thank you for fixing the issue!